pos_invoice_pay/models.py

```diff
@@ -66,9 +67,15 @@ def process_invoices_creation(self, sale_order_id):
 class AccountPayment(models.Model):
     _inherit = "account.payment"
 
-    paid_by_pos = fields.Boolean(default=False)
+    paid_by_pos = fields.Boolean(compute="_compute_paid_by_pos", store=True)
```

Comment:
Если поле не используется для поиска\фильтрации, то не обязательно сохранять в таблице значения
```suggestion
    paid_by_pos = fields.Boolean(compute="_compute_paid_by_pos")
```

Reply:
Это поле используется для поиска
https://github.com/itpp-labs/pos-addons/blob/8b16c4d146bed3c179c324fb604c758851da70a8/pos_invoice_pay/report/report.py#L33-L39

Reply:
Может напрямую по новому полю искать?

Reply:
> Может напрямую по новому полю искать?

Действительно можно, но дело в другом. Если какой-то сторонний модуль зависит от этого модуля и использует paid_by_pos, который изначально в базе хранится, то этот сторонний модуль сломается.

Reply:
Может сторонний модуль сделает store=True ?